crates/re_viewer/src/app.rs

```diff
@@ -562,6 +566,33 @@ impl App {
         }
     }
 
+    #[cfg(target_arch = "wasm32")]
+    fn run_copy_direct_link_command(&mut self, store_context: Option<&StoreContext<'_>>) {
+        let Some(SmartChannelSource::RrdHttpStream { url }) = store_context
+            .and_then(|ctx| ctx.recording)
+            .and_then(|rec| rec.data_source.as_ref())
+        else {
+            self.toasts.add(toasts::Toast {
+                kind: toasts::ToastKind::Warning,
+                text: format!("Could not copy direct link, no recording is open"),
+                options: toasts::ToastOptions::with_ttl_in_seconds(4.0),
+            });
+            return;
```

Comment:
Should this really be an error? I would expect to just get a copy to the viewer in this case (`https://app.rerun.io`)

Reply:
Btw, anything logged with `re_log::warn!` will also show up as a warning toast

Reply:
You're right, changed it to default to `location.origin` by itself